file/1.5/modules/ROOT/pages/file-documentation.adoc

```diff
@@ -458,7 +458,7 @@ A matcher can also be used for additional filtering of files.
 * <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
 | Auto Delete a| Boolean |  Whether each file should be deleted after processing or not. |  false |
 | Move To Directory a| String |  If provided, each processed file will be moved to a directory pointed by this path. |  |
-| Rename To a| String |  This parameter works in tandem with *Move to directory*. Use this parameter to enter the name under which the file should be moved. Do not set this parameter if moveToDirectory hasn't been set as well. |  |
+| Rename To a| String | This parameter works in tandem with *Move to directory*. Use this parameter to enter the name under which the file should be moved. Do not set this parameter if moveToDirectory hasn't been set as well. |  |
```

Comment:
You don't have this last sentence in an earlier description of this attribute. I don't think it's necessary.

Reply:
This is the generated reference. I didn't add it.